src/common/logger.go

```diff
@@ -0,0 +1,44 @@
+package common
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"path"
+
+	"github.com/urfave/cli/v2"
+)
+
+// the current slog does not support enable/disable feature, might need a wrap-arround struct
+var CgTopLogger slog.Logger
```

Comment:
Nothing in common should know about subsystems such as control groups.  How about just TopLogger, and subsystems can create their own loggers off of that?